starkbank/dynamicbrcode/__dynamicbrcode.py

```diff
@@ -0,0 +1,128 @@
+from ..utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date, check_timedelta
+
+
+class DynamicBrcode(Resource):
+    """# DynamicBrcode object
+    BR codes store information represented by Pix QR Codes, which are used to
+    send or receive Pix transactions in a convenient way.
+    When you initialize a DynamicBrcode, the entity will not be automatically
+    sent to the Stark Bank API. The 'create' function sends the objects
+    to the Stark Bank API and returns the list of created objects.
+    ## Parameters (required):
+    - amount [integer]: DynamicBrcode value in cents. Minimum = 1 (any value will be accepted). ex: 1234 (= R$ 12.34)
+    ## Parameters (optional):
+    - expiration [integer or datetime.timedelta, default 5097600 (59 days)]: time interval in seconds between due date and expiration date. ex 123456789
+    - tags [list of strings]: list of strings for tagging
```

Comment:
list of strings for tagging, will be passed to the reference Deposit resource when payed